build/flatpak/element.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+env TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID}" /app/element/element-desktop --no-sandbox "$@"
```

Comment:
I'm guessing the `--no-sandbox` is due to not being able to run as root? I'm afraid this isn't an option: electron needs this to isolate contexts so this is really important for security.